_docker/drupal/drupal-filesystem/web/themes/custom/rhdp/rhdp.theme

```diff
@@ -611,3 +611,31 @@ function rhdp_preprocess_views_view_grouping__product_groups(&$variables) {
     $variables['title']
   );
 }
+
+/**
+ * Implements hook_library_info_alter().
+ *
+ * By default, we will load the Keycloak prod script. If we've specified a
+ * different value in this environment's rhd.settings.yml file, we will alter
+ * that library definition here.
+ */
+function rhdp_library_info_alter(array &$libraries, $extension) {
+  $keycloak_url = \Drupal::config('redhat_developers')->get('keycloak')['scriptUrl'];
```

Comment:
@jordanpagewhite I think we should only do further processing here _if_ the user has provided an override through the the `rhd.settings.yml`, otherwise I think we can safely default to prod Keycloak JS URL in all environments. The only environment where we will override this is in `stage`, everything else will use the prod JS location.